examples/mnist/notebook.jl

```diff
@@ -129,7 +130,7 @@ fit!(mach, rows=1:500);
 # Computing an out-of-sample estimate of the loss:
 
 predicted_labels = predict(mach, rows=501:1000);
-cross_entropy(predicted_labels, labels[501:1000]) |> mean
+cross_entropy(predicted_labels, labels[501:1000])
```

Comment:
I do personally prefer training and testing data to be separate data structures (or at least to make this more generic by deciding a split size initially that could be changed as needed).

It may confuse some that we are loading the training data of MNIST then using the first 500 examples for actual training and the second 500 for testing/validation.

Reply:
Addressed.